crates/wascc-provider/src/lib.rs

```diff
@@ -214,6 +219,53 @@ impl<S: Store + Send + Sync> Provider for WasccProvider<S> {
         let client = kube::Client::new(self.kubeconfig.clone());
         let volumes = Ref::volumes_from_pod(&self.volume_path, &pod, &client).await?;
         for container in pod.containers() {
+            //switch code to here, before env var gets set
+            let mut port_assigned: i32 = 0;
+            if let Some(container_vec) = container.ports.as_ref() {
+                for c_port in container_vec.iter() {
+                    let container_port = c_port.container_port;
+                    let host_port = c_port.host_port;
+                    //self.port_set.lock().unwrap().insert(30000);
+                    if c_port.host_port.is_none() {
+                        println!("host port is not specified");
```

Comment:
can you review these `println` statements and see if they would be worth turning into `debug!` calls?